tree/ntuple/v7/doc/specifications.md

```diff
@@ -74,36 +103,63 @@ The meta-data envelope defines additional basic types (see below).
 
 ### Feature Flags
 
-Feature flags are 64bit integers where every bit represents a certain feature that is used
+Feature flags are 64bit integers where every bit represents a certain forward-incompatible feature that is used
```

Comment:
maybe we want to define what backward and forward-(in)compatible mean for us. My understanding is backward-compatible means "can read files written by previous versions" -- from a "consumer" point of view.

Then, does "forward-incompatible" mean "can't be read by future versions"? That doesn't sound right

Reply:
Backwards-compatible: new software version can read data produced by old software version
Forward-compatible: old software version can read data produced by new software version